auto/cc/test

```diff
@@ -70,7 +70,7 @@ case $NXT_CC_NAME in
 
         NXT_CFLAGS="$NXT_CFLAGS -O"
         #NXT_CFLAGS="$NXT_CFLAGS -O0"
-        NXT_CFLAGS="$NXT_CFLAGS -W -Wall -Wextra"
```

Comment:
In 17806fe5 ("Remove -W from compiler flags"), you can drop a Cc repeated field.